runtime/block_reduction.cu

```diff
@@ -15,46 +15,38 @@
 //  blockReduceSum<X_THREADS, Y_THREADS, Z_THREADS>
 //    (output[output_index], inputs[input_index],
 //      [] __device__ (T& a, const T b) { a += b; });
-//
```

Comment:
In addition to adding `aligned` template parameter, I also removed code that was only necessary for ROCM compatibility.